raster-foundry/raster-foundry
# Filter out most recent MODIS / Landsat 7 images

## Overview

This PR prevents maxAcquisitionDatetimes more recent than 24 hours ago from being sent off to the NASA CMR Earthdata API.

### Checklist


### Notes

It's a bandaid :man_shrugging: 

## Testing Instructions

- wander over to staging to the v2 ui, browse for imagery from NASA CMR in Landsat 7 or MODIS
- check out that sweet warning
- set the max date to at least a day ago
- check out that the warning went away

Closes #4869
